index.js

```diff
@@ -335,6 +336,8 @@ slackMessages.action('confirm', (payload, respond) => {
                 logError(error);
               });
             });
+
+            newIncidentNotifier(result.data.data);
```

Comment:
Now here you can add a then n catch clauses that will ensure that we catch log any error or know that the operation was successgul